ci-lib-core/src/sql.rs

```diff
@@ -245,6 +245,7 @@ pub const CREATE_RUNS_TABLE: &'static str = "\
         job_id INTEGER,
         artifacts_path TEXT,
         state INTEGER NOT NULL,
+        host_preference INTEGER,
```

Comment:
i seem to have `host_preference` as TEXT rather than INTEGER. the text in that column is an integer key into `hosts` though, so i think this is right and i'm not sure why my current deployment is correct.

Reply:
i think SQLite is... untyped? or dynamic typed? so if the Rust calls and parses it as an INTEGER everwhere, it might not matter what it is in the database? but i also might be terribly wrong.